redash/query_runner/oracle.py

```diff
@@ -66,7 +66,7 @@ def configuration_schema(cls):
             "properties": {
                 "user": {"type": "string"},
                 "password": {"type": "string"},
-                "host": {"type": "string"},
+                "host": {"type": "string", "title": "Host (When type _donotmakedsn, DSN Service Name is used instead.)"},
```

Comment:
For the text, I think this will sound more "natural":

    To use a DSN Service Name instead, use the text string `_donotmakedsn` in the host name field.

Hmmm.  Reading that back though, the special name of "do not make dsn" is confusing when you want to make a ***DSN*** Service Name.

Reply:
Would an option name like `_useservicename` be clearer?

Reply:
Thank you.

I'll try to rewrite it.

Reply:
Thank you.

You're right, it's not a good name.
Agreed.

Reply:
>Would an option name like _useservicename be clearer?

I think this name is easier to understand the meaning.

Reply:
Cool. :smile: